bin/blade-icons-generate

```diff
@@ -6,7 +6,12 @@ use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Console\SingleCommandApplication as Command;
 
-require __DIR__.'/../vendor/autoload.php';
+$autoloadPath = getcwd() . "/vendor/autoload.php";
```

Comment:
For me doing this version of the code it works fine with either VCS (realistic to composer installs) and path based repos. So I think this would make more sense in the end?

Reply:
I don't understand why this is needed. Can you explain a bit more extensively?

Reply:
See the replies here:
https://github.com/blade-ui-kit/blade-icons/pull/144#discussion_r657953501
https://github.com/blade-ui-kit/blade-icons/pull/144#discussion_r657958061

The way you're locally developing things is skewing what files should and shouldn't exist for end users. If you were to use `vcs` instead of `path` repos in composer:

```
        {
            "type": "vcs",
            "url": "https://github.com/mallardduck/blade-icons"
        }
```

Then you can test only the changes you made to refactor with:
```
        "blade-ui-kit/blade-icons": "dev-generator-dries",
```

And compare to:
```
        "blade-ui-kit/blade-icons": "dev-abstract-generator-command",
```